tests/parser/functions/test_minmax.py

```diff
@@ -185,3 +185,29 @@ def foo() -> uint256:
         lambda: get_contract_with_gas_estimation(code_2),
         TypeMismatchException
     )
+
+
+def test_signed_unsigned(get_contract_with_gas_estimation):
+    code = """
+@public
+def foo1() -> uint256:
+    return min(0, 2**255)
+
+@public
+def foo2() -> uint256:
+    return min(2**255, 0)
+
+@public
+def foo3() -> uint256:
+    return max(0, 2**255)
+
+@public
+def foo4() -> uint256:
+    return max(2**255, 0)
```

Comment:
Can we do `2**127` and `-2**127` in both positions? Something that exercises the signed logic